vizro-core/src/vizro/models/_components/form/range_slider.py

```diff
@@ -131,3 +128,20 @@ def update_slider_values(start, end, slider, input_store):
             className="selector_container",
             id=f"{self.id}_outer",
         )
+
+    def _update_slider_values(self, trigger_id, start, end, slider, input_store, value):
```

Comment:
Mea culpa, I think its better to rename start and end to start_txt and end_txt, correct right?